addons/account_asset/models/account_asset.py

```diff
@@ -103,6 +112,21 @@ class AccountAssetAsset(models.Model):
         help="It is the amount you plan to have that you cannot depreciate.")
     invoice_id = fields.Many2one('account.invoice', string='Invoice', states={'draft': [('readonly', False)]}, copy=False)
     type = fields.Selection(related="category_id.type", string='Type', required=True)
+    date_first_depreciation = fields.Selection([
+        ('first_day_month', 'Purchase Date'),
```

Comment:
the rest of the help tooltip might be improved too for manual (see my first comment)